pkg/apis/pipeline/v1alpha1/git_resource.go

```diff
@@ -25,7 +25,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-const workspaceDir = "/workspace"
+const WorkspaceDir = "/workspace"
```

Comment:
hm too bad this had to change but worth it i guess

Reply:
yeah, iirc certain fields and methods(?) had to be exported in order for the package separation to work